syntax = "proto3";

option go_package = "sfc-virtual-device-plugin/efvi";
package efvi;

message ApplyRequest {
  string host = 1;
  int32 port = 2;
  string interface = 3;
  string name = 4;
}

message ViResource {
  int32 code = 1;
  string message = 2;
  string viPath = 3;
  string queuePath = 4;
}

service EfviService {
  rpc ApplyVirtualDevice(ApplyRequest) returns (ViResource) {}
} 